templates/openstackconfiggenerator/bin/create-playbooks.sh

```diff
@@ -57,8 +57,12 @@ sed -e "s|/usr/share/openstack\-tripleo\-heat\-templates|\.|" -i $HOME/config-tm
 cp -a $HOME/config-tmp/* "$TEMPLATES_DIR/"
 
 pushd $TEMPLATES_DIR
-python3 tools/process-templates.py -r $TEMPLATES_DIR/roles_data.yaml -n $TEMPLATES_DIR/network_data.yaml
 
+# Remove unused roles from roles_data.yaml as container image prepate skips roles with count=0 but
```

Comment:
this is really good, as it also fixes the situation where you copy a default role, which might not be used, but has a default count >0. That results in an error as you don't have a bmset or vmset for it.